packages/devtools_app/lib/src/screens/memory/panes/profile/profile_pane_controller.dart

```diff
@@ -10,24 +10,38 @@ import 'package:vm_service/vm_service.dart';
 
 import '../../../../shared/config_specific/import_export/import_export.dart';
 import '../../../../shared/globals.dart';
+import '../../../../shared/primitives/simple_items.dart';
 import '../../shared/heap/class_filter.dart';
 import 'model.dart';
 
+class _Json {
+  static const profile = 'profile';
+}
```

Comment:
use enum or just a private static variable since there is only one field. same comment elsewhere